roles/change-paths/tasks/main.yml

```diff
@@ -205,3 +205,9 @@
 - name: Save modified JS Injector rules to disk
   shell: "{{ drush_alias }} scr {{ scripts_dir }}/js-injector.php"
   notify: Clear site cache
+
+# Bring 'er out of hyperspace, Chewy
```

Comment:
This is not really a change-paths task. Yes, it is in the correct place for the order of operations but shouldn't we move it to a better-named role/task?

Reply:
Fair 'nuff. Let me noodle on that.